babel-plugin-code-push/index.js

```diff
@@ -0,0 +1,198 @@
+const path = require("path");
+const fs = require("fs");
+const { parseExpression, parse } = require("@babel/parser");
+
+const OPTIONS_TO_BUNDLE = [
+  "bundleHost",
+  "runtimeVersion",
+  "versioning",
+  "updateChecker",
+];
```

Comment:
1\.
I think `bundleHost` might become an unnecessary option.
But there’s no need to remove it now.

2\.
I’m concerned that writing the `updateChecker` function in the config file might not provide a good developer experience.
Additionally, unlike in the CLI, the Runtime may require tasks such as bug tracking or logging before/after the `updateChecker` function.
The runtime library also needs a function to determine whether the current app platform is iOS or Android.

It might be better to share only the resource path/endpoint for retrieving information, rather than sharing the implementation of the `updateChecker`.

--

I’d like to discuss these configurations in more detail later.  
For now, let’s focus on implementing the features and tidying things up as we progress. 🦾